### DISCUSSION TITLE
Unicode arrow in code?

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#88] [General]
I see U+2192 in this code:

```
class Hello {
  function : Main(args : String[]) ~ Nil {
    "Hello World" → PrintLine();
    "Καλημέρα κόσμε" → PrintLine();
    "こんにちは 世界" → PrintLine();
  }
}

```

It's not the simple -> of C++.

**Comment by objeck** [comment]
Objeck supports Unicode and non-Unicode operations. It also internally supports Unicode strings. 

The language syntax supports the following Unicode operations:

- "->" as  '→'
- ":=" as  '←'
- "<=" as  '≤'
- ">=" as  '≥'

**Reply by ghost** [reply]
Unicode operators? It's a terrible idea.

**Reply by objeck** [reply]
Optional feature

**Reply by ghost** [reply]
https://www.emojicode.org/

**Reply by ghost** [reply]
An example of over abusing Unicode.

**Reply by ghost** [reply]
https://www.dcs.gla.ac.uk/%7Ewpc/reports/compilers/compilerindex/VPUnicode.htm

**Reply by ghost** [reply]
Do you want the code in your language look like that?
